.github/workflows/main.yml

```diff
@@ -69,6 +69,10 @@ jobs:
           python: '3.11'
           allow_failure: false
 
+        - name: py311-dj50-postgres-xdist-coverage
+          python: '3.11'
+          allow_failure: false
+
```

Comment:
I couldn’t figure out the logic for what is on the test matrix here, so I only added one entry.

This can be expanded to Python 3.12 with #1086.